packages/e2e_tests/cypress/e2e/get-updates-sign-up.cy.js

```diff
@@ -0,0 +1,35 @@
+import { PO_GetUpdates } from '../pageObject/Get-updates/PO_GetUpdates';
+
+const getUpdates = new PO_GetUpdates();
+
+describe('User subscribes to get updates about a project', () => {
+	it('User navigaate to the Get Updates start page', () => {
```

Comment:
```suggestion
	it('should navigate to the Get Updates start page', () => {
```

Reply:
And the it blocks, "it should ...."

Reply:
I have fixed this and added should keyword